packages/camera/camera/lib/camera.dart

```diff
@@ -2,643 +2,8 @@
-    if (!value.isInitialized || _isDisposed) {
-      throw CameraException(
-        'Uninitialized CameraController',
-        'stopVideoRecording was called on uninitialized CameraController',
-      );
-    }
-    if (!value.isRecordingVideo) {
-      throw CameraException(
-        'No video is recording',
-        'stopVideoRecording was called when no video is recording.',
-      );
-    }
-    try {
-      value = value.copyWith(isRecordingVideo: false);
-      await _channel.invokeMethod<void>(
-        'stopVideoRecording',
-        <String, dynamic>{'textureId': _textureId},
-      );
-    } on PlatformException catch (e) {
-      throw CameraException(e.code, e.message);
-    }
-  }
-
-  /// Pause video recording.
-  ///
-  /// This feature is only available on iOS and Android sdk 24+.
-  Future<void> pauseVideoRecording() async {
-    if (!value.isInitialized || _isDisposed) {
-      throw CameraException(
-        'Uninitialized CameraController',
-        'pauseVideoRecording was called on uninitialized CameraController',
-      );
-    }
-    if (!value.isRecordingVideo) {
-      throw CameraException(
-        'No video is recording',
-        'pauseVideoRecording was called when no video is recording.',
-      );
-    }
-    try {
-      value = value.copyWith(isRecordingPaused: true);
-      await _channel.invokeMethod<void>(
-        'pauseVideoRecording',
-        <String, dynamic>{'textureId': _textureId},
-      );
-    } on PlatformException catch (e) {
-      throw CameraException(e.code, e.message);
-    }
-  }
-
-  /// Resume video recording after pausing.
-  ///
-  /// This feature is only available on iOS and Android sdk 24+.
-  Future<void> resumeVideoRecording() async {
-    if (!value.isInitialized || _isDisposed) {
-      throw CameraException(
-        'Uninitialized CameraController',
-        'resumeVideoRecording was called on uninitialized CameraController',
-      );
-    }
-    if (!value.isRecordingVideo) {
-      throw CameraException(
-        'No video is recording',
-        'resumeVideoRecording was called when no video is recording.',
-      );
-    }
-    try {
-      value = value.copyWith(isRecordingPaused: false);
-      await _channel.invokeMethod<void>(
-        'resumeVideoRecording',
-        <String, dynamic>{'textureId': _textureId},
-      );
-    } on PlatformException catch (e) {
-      throw CameraException(e.code, e.message);
-    }
-  }
-
-  /// Releases the resources of this camera.
-  @override
-  Future<void> dispose() async {
-    if (_isDisposed) {
-      return;
-    }
-    _isDisposed = true;
-    super.dispose();
-    if (_creatingCompleter != null) {
-      await _creatingCompleter.future;
-      await _channel.invokeMethod<void>(
-        'dispose',
-        <String, dynamic>{'textureId': _textureId},
-      );
-      await _eventSubscription?.cancel();
-    }
-  }
-}
+export 'package:camera_platform_interface/camera_platform_interface.dart';
```

Comment:
I seem to have some problems with this one. Locally I have updated this statement to:

```dart
export 'package:camera_platform_interface/camera_platform_interface.dart'
    show
        CameraDescription,
        CameraException,
        CameraLensDirection,
        ResolutionPreset,
        XFile;
```

But now the code in the example/lib/main.dart won't compile anymore because it can't locate any of the types mentioned by the `show` statement. I made sure I imported the `import package:camera/camera.dart` Dart file but still not working.

I can't figure out what I am doing wrong here, probably overlooking something (it is getting late here). Maybe you have some pointers? Will give it a fresh look tomorrow morning.

Reply:
It should work, maybe you need a flutter clean in the example app, or to "restart" the analyzer (or both?).

If you want, push whatever you have to the branch and I can take a look on my end (it's going to break some tests, but oh well!)

Reply:
it was indeed my environment that got a bit confused ;) I corrected everything and it works without issues now.